Functions.Templates/Templates/SqlInputBinding-JavaScript/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
```

Comment:
am I reading correctly that the input binding template doesn't have a trigger?
can we be consistent with the C# templates, and add
```javascript
content.res = {
body: results,
// header for the mimetype
}
```

Reply:
returned the results for both the bindings now.

Reply:
I'm still not seeing a trigger binding for this - we should add an HTTP trigger binding to match the C# templates